homeassistant/components/vesync/fan.py

```diff
@@ -107,6 +111,7 @@ class VeSyncFanHA(VeSyncBaseEntity, FanEntity):
     )
     _attr_name = None
     _attr_translation_key = "vesync"
+    _enable_turn_on_off_backwards_compatibility = False
```

Comment:
What does this do ?

Reply:
Not sure - was in the other PR. I have removed for now.